Cargo.toml

```diff
@@ -46,6 +46,9 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tokio = { version = "1", features = ["full"] }
 
+[target.'cfg(not(target_os = "windows"))'.dependencies]
+openssl = { version = "0.10", features = ["vendored"] }
+
```

Comment:
@imunproductive @oleksandrvoyager Do we really need OpenSSL on Windows? The idea was to use RustTLS, if you'll recall